cases/transaction.test.js

```diff
@@ -2,93 +2,137 @@ import { fetch } from "./util/fetchShim";
 import getSep10Token from "./util/sep10";
 import getTomlFile from "./util/getTomlFile";
 import StellarSDK from "stellar-sdk";
-import FormData from "form-data";
-import { transactionSchema } from "./util/schema";
+import { createTransaction } from "./util/transactions";
+import {
+    errorSchema,
+    getTransactionSchema
+} from "./util/schema";
+
+jest.setTimeout(60000);
 
 const urlBuilder = new URL(process.env.DOMAIN);
 const domain = urlBuilder.toString();
-const account = "GCQJX6WGG7SSFU2RBO5QANTFXY7C5GTTFJDCBAAO42JCCFIMZ7PEBURP";
 const secret = "SAUOSXXF7ZDO5PKHRFR445DRKZ66Q5HIM2HIPQGWBTUKJZQAOP3VGH3L";
 const keyPair = StellarSDK.Keypair.fromSecret(secret);
 
 describe("Transaction", () => {
-  let toml;
-  let enabledCurrency;
-  let jwt;
-
-  const createTransaction = async (asset_code, account, authenticate) => {
-    const params = new FormData();
-    if (asset_code) params.append("asset_code", asset_code);
-    if (account) params.append("account", account);
-    const authenticatedHeaders = Object.assign(
-      {
-        Authorization: `Bearer ${jwt}`
-      },
-      params.getHeaders()
-    );
-    const response = await fetch(
-      toml.TRANSFER_SERVER + "/transactions/deposit/interactive",
-      {
-        headers: authenticate ? authenticatedHeaders : params.getHeaders(),
-        method: "POST",
-        body: params
-      }
-    );
-    const status = response.status;
-    const json = await response.json();
-    return {
-      status,
-      json
-    };
-  };
-  beforeAll(async () => {
-    toml = await getTomlFile(domain);
-    jwt = await getSep10Token(domain, keyPair);
-    const infoResponse = await fetch(toml.TRANSFER_SERVER + "/info", {
-      headers: {
-        Origin: "https://www.website.com"
-      }
+    let toml;
+    let enabledCurrency;
+    let jwt;
+
+    beforeAll(async () => {
+        toml = await getTomlFile(domain);
+        jwt = await getSep10Token(domain, keyPair);
+        const infoResponse = await fetch(toml.TRANSFER_SERVER + "/info", {
+            headers: {
+                Origin: "https://www.website.com"
+            }
+        });
+        const infoJSON = await infoResponse.json();
+        const currencies = Object.keys(infoJSON.deposit);
+
+        enabledCurrency = currencies.find(
+            currency => infoJSON.deposit[currency].enabled
+        );
+
+        expect(toml.TRANSFER_SERVER).toBeDefined();
+    });
+
+    it("has CORS on the transaction endpoint", async () => {
```

Comment:
This is hitting the `/fee` endpoint, not the `/transaction` endpoint. `/fee` may also require authentication (Polaris does), so this could fail for anchor servers with `fee.authentication_required`